bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -446,30 +439,30 @@ spec:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: RELATED_IMAGE_BACKEND
-                  value: quay.io/3scale/apisonator:latest
+                  value: quay.io/3scale/3scale214:apisonator-3scale-2.14.1-GA
                 - name: RELATED_IMAGE_APICAST
-                  value: quay.io/3scale/apicast:latest
+                  value: quay.io/3scale/3scale214:apicast-3scale-2.14.1-GA
                 - name: RELATED_IMAGE_SYSTEM
-                  value: quay.io/3scale/porta:latest
+                  value: quay.io/3scale/3scale214:porta-3scale-2.14.1-GA
                 - name: RELATED_IMAGE_ZYNC
-                  value: quay.io/3scale/zync:latest
+                  value: quay.io/3scale/3scale214:zync-3scale-2.14.1-GA
                 - name: RELATED_IMAGE_SYSTEM_MEMCACHED
                   value: memcached:1.5
                 - name: RELATED_IMAGE_BACKEND_REDIS
-                  value: quay.io/centos7/redis-6-centos7:latest
+                  value: quay.io/fedora/redis-6:latest
                 - name: RELATED_IMAGE_SYSTEM_REDIS
-                  value: quay.io/centos7/redis-6-centos7:latest
+                  value: quay.io/fedora/redis-6:latest
                 - name: RELATED_IMAGE_SYSTEM_MYSQL
-                  value: centos/mysql-80-centos7
+                  value: quay.io/sclorg/mysql-80-c8s
                 - name: RELATED_IMAGE_SYSTEM_POSTGRESQL
-                  value: centos/postgresql-10-centos7
+                  value: quay.io/sclorg/postgresql-10-c8s
                 - name: RELATED_IMAGE_ZYNC_POSTGRESQL
-                  value: centos/postgresql-10-centos7
+                  value: quay.io/sclorg/postgresql-10-c8s
                 - name: RELATED_IMAGE_OC_CLI
                   value: quay.io/openshift/origin-cli:4.7
                 - name: RELATED_IMAGE_SYSTEM_SEARCHD
                   value: quay.io/3scale/searchd:latest
```

Comment:
Should this be latest?

Reply:
Thanks for comment. What you sugegst @MStokluska ?

Reply:
he is referring to `quay.io/3scale/searchd:latest`. I also spotted that.